### PR TITLE
IDE guide

### PR DESCRIPTION
**Pull request by delagroove**

### CODE REVIEW

**Comment by brent-hoover** [comment]
@delagroove This is a good start but let's talk about how we can flesh this out and make it more helpful.

File: docs/setting-your-IDE.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
+## The basics
+
+If you are interested in being able to quickly debug and develop Mailchimp Open Commerce, consider installing these useful tools to help you.
+
+- You should use [nvm](https://github.com/nvm-sh/nvm) to install and manage NodeJS.
+- Visual Studio Code is a preferred text editor but you can use Sublime Text, Emacs, Vim or any other editor of your choice.
+- [Kadira](https://github.com/kadira-open/kadira-server) can help you optimize performance of a Meteor app.
```
Comment (Akarshit) [inline docs/setting-your-IDE.md]: Let's remove Kadira. Also it's only needed for meteor app, which at this point is only Admin.

**Review by Akarshit — CHANGES_REQUESTED** [review]

File: docs/setting-your-IDE.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
+## The basics
+
+If you are interested in being able to quickly debug and develop Mailchimp Open Commerce, consider installing these useful tools to help you.
+
+- You should use [nvm](https://github.com/nvm-sh/nvm) to install and manage NodeJS.
+- Visual Studio Code is a preferred text editor but you can use Sublime Text, Emacs, Vim or any other editor of your choice.
```
Comment (brent-hoover) [inline docs/setting-your-IDE.md:6]: Let's add Webstorm to this list

File: docs/setting-your-IDE.md [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+
+## Browser Extensions
+
+### Chrome
+
+- [React Developer Tools](https://chrome.google.com/webstore/detail/react-developer-tools/fmkadmapgofadopljbjfkapdkoienihi)
```
Comment (brent-hoover) [inline docs/setting-your-IDE.md:12]: I think this list should add some context as to why you would want to install these and we might want to break these down a little bit into types of extensions.

File: docs/setting-your-IDE.md [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+- [z-context](https://chrome.google.com/webstore/detail/z-context/jigamimbjojkdgnlldajknogfgncplbh)
+
+### Firefox
+
+- [Altair GraphQL Client](https://addons.mozilla.org/en-US/firefox/addon/altair-graphql-client/)
+- [React Developer Tools](https://addons.mozilla.org/en-US/firefox/addon/react-devtools/)
+
+## Code Editor Extensions
+
+### [Visual Studio Code](https://code.visualstudio.com/)
```
Comment (brent-hoover) [inline docs/setting-your-IDE.md:26]: Let's limit these to ones that are more MOC specific and add some info on why you should add them. I think just a list of plugins is not going to be that helpful.

File: docs/setting-your-IDE.md [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+- [GraphiQL Standalone App](https://github.com/skevy/graphiql-app)
+
+## Troubleshooting
+due to misconfiguration, missing dependencies, operating system differences, and software bugs. Here are some tips for diagnosing and fixing these issues.
+
+### Docker Issues
+
+These are potential issues you might encounter when running Reaction within a local Docker environment using Docker Compose and the docker-compose.yml file.
+
+### Memory errors or errors about "Meteor rawLogs"
```
Comment (brent-hoover) [inline docs/setting-your-IDE.md:88]: Not sure how this section is related to setting up your IDE?

**Review by brent-hoover — CHANGES_REQUESTED** [review]
Let's have fewer items listed with more info about why you should install them and maybe move the Troubleshooting stuff somewhere else?

File: docs/setting-your-IDE.md [hunk 6]
```diff
@@ -0,0 +1,91 @@
...
+- [GraphQL Playground](https://github.com/prismagraphql/graphql-playground)
+- [Altair GraphQL Client](https://altair.sirmuel.design/)
+- [GraphiQL Standalone App](https://github.com/skevy/graphiql-app)
+
+## Troubleshooting
+due to misconfiguration, missing dependencies, operating system differences, and software bugs. Here are some tips for diagnosing and fixing these issues.
+
+### Docker Issues
+
+These are potential issues you might encounter when running Reaction within a local Docker environment using Docker Compose and the docker-compose.yml file.
```
Comment (MohanNarayana) [inline docs/setting-your-IDE.md:86]: Name reaction to Open Commerce

File: docs/setting-your-IDE.md [hunk 7]
```diff
@@ -0,0 +1,91 @@
...
+- [Studio 3T or Robo 3T](https://robomongo.org/)
+
+## Standalone GraphQL Clients
+
+- [GraphQL Playground](https://github.com/prismagraphql/graphql-playground)
+- [Altair GraphQL Client](https://altair.sirmuel.design/)
+- [GraphiQL Standalone App](https://github.com/skevy/graphiql-app)
+
+## Troubleshooting
+due to misconfiguration, missing dependencies, operating system differences, and software bugs. Here are some tips for diagnosing and fixing these issues.
```
Comment (MohanNarayana) [inline docs/setting-your-IDE.md:82]: Capitalize 'Due'

**Comment by brent-hoover** [comment]
Closing to be reopened in a later phase

**Pull request closed without merging**